src/test/java/Homework20.java

```diff
@@ -0,0 +1,51 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework20 extends BaseTest {
+
+
+    @Test
+    public void deletePlaylist() throws InterruptedException{
+
+        String playlistDeletedNotification = "Deleted playlist \"Sanjeelas Playlist1.\"";
+
+        navigateToLoginPage();
+        provideEmail("sanjeela.chitrakar@testpro.io");
+        providePassword("te$t$tudent1");
+        clickSubmit();
+        clickPlaylist();
+
+
+        clickDeleteThisPlaylist();
+
+
+        Assert.assertEquals(getDeletePlaylistNotification(), playlistDeletedNotification);
+
+
+    }
+
+    public void clickPlaylist(){
+        WebElement playlist = wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//section[@id='playlists']//a[contains(text(),'Sanjeelas Playlist1')]")));
+        playlist.click();
+
+    }
+
+    public void clickDeleteThisPlaylist() throws InterruptedException{
+        WebElement deleteThisPlaylist = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("button[class='del btn-delete-playlist']")));
+        deleteThisPlaylist.click();
+
+        Thread.sleep(2000);
```

Comment:
Please, delete `throws InterruptedException`

Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.